synctl/cli.py

```diff
@@ -60,6 +60,7 @@
 SYN_ALERT = "alert"  # short for smart alerts
 SYN_RESULT = "result"
 POP_SIZE = "pop-size"
```

Comment:
I am wondering if we can change `pop-size` to `size` to make it simple as `cost`. @rui1234 what do you think?

Reply:
I prefer to use "pop-size" and "pop-cost" for consistency, because simple "size" or "cost" maybe confusing.

Other names like alert, result, test etc, people can understand them as synthetic alert, synthetic result, synthetic test, but for size and cost, they can not be thought as synthetic size or synthetic cost.

Reply:
OK.